.github/workflows/build.yaml

```diff
@@ -62,7 +62,7 @@ jobs:
               with:
                   php-version: ${{ matrix.php }}
             - run: |
-                  sed -ri 's/"symfony\/(.+)": "(.+)"/"symfony\/\1": "'${{ matrix.symfony }}'"/' composer.json;
+                  sed -ri '/symfony\/deprecation-contracts/!s/"symfony\/(.+)\symfony\/deprecation-contracts/;": "(.+)"/"symfony\/\1": "'${{ matrix.symfony }}'"/' composer.json;
```

Comment:
we need a space between `!` and `s`

Reply:
you added it before the wrong `s`...

Reply:
Wow, this was giving me a headache. Thanks for the charm.

![image](https://i1.wp.com/media.giphy.com/media/X9JE4J2qYDDO0/giphy.gif?resize=500%2C209&ssl=1)

Ready to merge?

![image](https://www.meme-arsenal.com/memes/bfac55617669972a4228be62a01f234b.jpg)